src/encoded/static/inline.js

```diff
@@ -16,6 +16,52 @@ if ('NodeList' in window && !NodeList.prototype.forEach) {
     };
 }
 
+// IE11 polyfill for [].fill, based on- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill#Polyfill
+if (!Array.prototype.fill) {
```

Comment:
Babel polyfills this already.

Reply:
Good catch. In my mind, babel is a transpiler. I got to remember it also polyfills